doc/users/dflt_style_changes.rst

```diff
@@ -1065,6 +1065,17 @@ Z-order
 
 
 
+``ScalarFormatter`` tick label formatting with offsets
+======================================================
+
+With the default of ``rcParams['axes.formatter.useoffset'] = True``,
+an offset will be used when it will save 4 or more digits.  This can
```

Comment:
Ugh, while that explanation is too long in docs, it's by far the clearest. Thanks!
So I think it's something like "Subtracts offset from ticks when ticks have more significant digits than offset_threshold to reduce tick label size to offset_threshold."